test/Project.test.js

```diff
@@ -551,4 +551,9 @@ describe("project", function() {
         expect(project.getLocaleInherit("ko-KR")).toBeUndefined();
         expect(project.getLocaleInherit()).toBeUndefined();
     });
+    test("GetProjectType", function() {
+        expect.assertions(1);
+        var project = ProjectFactory('./test/testfiles', {});
+        expect(project.getProjectType().toString()).toBe("web");
```

Comment:
Why do we need ".toString()" in there? The documentation says that `getProjectType()` returns a string already.

Reply:
I fixed it. and I also fixed other cases that are in same case